doctr/models/predictor/tensorflow.py

```diff
@@ -74,7 +74,11 @@ def __call__(
 
         # Localize text elements
         loc_preds = self.det_predictor(pages, **kwargs)
+
         # Crop images
+        if not isinstance(pages[0], np.ndarray):
+            pages = [page.numpy() for page in pages]
+
```

Comment:
Isn't that a detour? We convert it to numpy, but it will be converted back to TF/Pytorch tensor afterwards?

Reply:
the prepare_crops method needs numpy arrays

Reply:
We typed it as such, because it's in base but it's not exactly the case: it works with all sorts of tensors